### ISSUE TITLE
managers.tablemanagers.IslandTableManager.save()" is null?

### ISSUE THREAD

**Issue by infinitelow** [Iridium-Development/IridiumSkyblock#712]
**Describe the bug**
Plugin is disabled. 

**To Reproduce**
Change jars from 4.0.0 to 4.0.1

**Expected behavior**
Plugin to be enabled

**Server and Plugin Version**
 CraftBukkit version 3758-Spigot-7d7b241-1f2dd8e (MC: 1.19.4) (Implementing API version 1.19.4-R0.1-SNAPSHOT)
Plugin is disabled. So I can't

**Screenshots**

**Logs**
https://mclo.gs/ENHaaOE

**Additional context**
Server is in a Alpha state. 4.0.0 was working fine also. 

plugins

[INFO] Plugins (32): PlaceholderAPI, BuycraftX, OldCombatMechanics, DeluxeTags, AntiDonkeyDupe, ViaVersion, mcMMO, ViaBackwards, MiniMOTD, LuckPerms, ProtocolLib, Vault, StaffPlus, SimpleScore, DecentHolograms, Essentials, AdvancedBan, FastAsyncWorldEdit (WorldEdit), EssentialsChat, NexEngine, EssentialsSpawn, ExcellentCrates, CoreProtect, ShopGUIPlus, IridiumSkyblock, ItemEdit, IllegalStack, Multiverse-Core, WildStacker, JetsMinions, CrazyEnvoys, PlugManX (PlugMan)

**Comment by infinitelow** [comment]
1. Ok I would like to say I'm sorry I did not read, If you think you've found a bug, please make sure you isolate the issue down to IridiumSkyblock and its dependencies before posting an issue in our [Issues](https://github.com/Iridium-Development/IridiumSkyblock/issues) tab. While you're there, please follow our issues guidelines. , Until after I already made this. 

2. I don't know how to delete a issue like this. 

3.I joined the discord yesterday . I  deleted all this files of the 3. build for the 4.0.0 build. But I did not delete all the files for the 4.0.0 for the new 4.0.1 build.   I thought the 4.0.0 and the 4.0.1 were just kind of the same thing.  

Long story short : Answer 
You still need to delete all your config files with 4.0.0 if you want to use 4.0.1.

**Comment by PeachesMLG** [comment]
>  You still need to delete all your config files with 4.0.0 if you want to use 4.0.1.
This shouldnt be the case, ill look into this
